frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -45,31 +45,35 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
   const HelmActionConfig: { [helmAction: string]: HelmActionConfigType } = {
     [HelmActionType.Install]: {
       title: 'Install Helm Chart',
+      subTitle: 'The helm chart will be installed using the YAML shown in the editor below.',
       helmReleaseApi: `/api/helm/chart?url=${chartURL}`,
       fetch: coFetchJSON.post,
       redirectURL: `/topology/ns/${namespace}`,
     },
     [HelmActionType.Upgrade]: {
       title: 'Upgrade Helm Release',
+      subTitle: '',
       helmReleaseApi: `/api/helm/release?ns=${namespace}&release_name=${releaseName}`,
       fetch: coFetchJSON.put,
       redirectURL: `/helm-releases/ns/${namespace}`,
     },
   };
 
+  const context = HelmActionConfig[helmAction];
```

Comment:
There really is no reason to create a map of all config types when we only need one. There should be a util function `getHelmActionConfig` that uses a `switch` on the action to return only the config necessary.
If you use `useMemo` on the result you can also prevent recreating the config and use it as a dependency in other hooks.

Reply:
Even if I create a util function that returns the config based on the action, it will still have to be inside the render scope since the config is using variables from the scope. Or else I'll have to pass everything down as param.

Reply:
Moved the config to a function and using `useMemo` now.